fastly.toml

```diff
@@ -3,15 +3,3 @@ name = "Default starter for Rust"
 description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
 authors = ["<oss@fastly.com>"]
 language = "rust"
-
-# The [setup] section is used by the Fastly CLI to provide a more integrated experience for shared Starter Kits.
-# https://developer.fastly.com/reference/fastly-toml/
-#
-[setup.backends.backend_name]
-description = "A backend able to serve `/articles` path"
-address = "reqbin.com"
-port = 443
-[setup.backends.other_backend_name]
-description = "A backend able to serve `/anything` path"
-address = "httpbin.org"
-port = 443
```

Comment:
I think that omitting the [setup] section will make the CLI prompt the user to add resources, wheras adding an empty [setup] section would suppress that prompt.  Is that right @Integralist ?  If so, I think it's the right call to omit entirely, because with these super generic starters, it's likely that the user will modify the code before deploying in a way that will add a resource requirement.

Reply:
So the first thing to clarify is that the CLI will only utilise `[setup]` if we're dealing with a project that has no existing service. Additionally the only `[setup]` resource that the CLI will prompt the user for even when that resource isn't defined in `[setup]` is a backend as we expect at least one backend to exist.

Reference:  
https://github.com/fastly/cli/blob/main/pkg/commands/compute/deploy.go#L200-L209